### ISSUE TITLE
Issues running PSA addon on render farm (aggregation of issues)

### ISSUE THREAD

**Issue by karlisup** [PhysicalAddons/physical-starlight-and-atmosphere#37]
Ticket 1
[..] Another thing I'd like to see is renderfarm mode. Currently, the addon updates the material every frame as the addon settings change with keyframes. You add a way to bake keyframes directly into the sun lamp and the material for use with renderfarms (e.g. SheepIt) that block scripts.

Ticket 2
[..] Wondering if there are any secrets to uploading the addon's animation to a render farm like render.st... I am getting the Physical Starlight and Atmosphere to register with the project, but all of the keyframes I inserted (MIE scattering/RGB/Elevation) seem to be left out. I contacted render.st and they said they've had trouble before doing the same thing... [..]

Ticket 3
[..] I’m trying to export a project for a render farm that involves rising and setting sun. Whole sun animation is done with physical starlight and atmosphere addon. When I render it in render farm sun doesn't animate and the whole animation appears just black as a 1st frame. I was told by support that I need to pack this addons animation cache file with the project, but I can’t find a way to do it. [..]

**Comment by hnoerenberg** [comment]
Question about ticket no 1: Is there no chance to run PSA on a farm at all? 1.3.1 is even not rendering when I start the render over a local small deadline farm. The Material is not changing and the keyframes are being ignored. 
Is there a possibility somehow?

**Comment by karlisup** [comment]
I got great input from Stephen Leger which is the Archipack developer explaining what he does for his addon in order to support render farms. We'll prioritize this after the v1.4.2 stable release.

**Comment by markrmiller** [comment]
Are people able to get this to work with the presets without key framing it? I use one of the earth presets and I cannot get it to render anything but a black sky on other machines. Doesn't matter if I pack the blend file, change to relative paths and put the preset blend file on the other machines. Nothing seems to do it.

**Comment by karlisup** [comment]
Hey @markrmiller,
Do you have access to the console? Do you get any errors? 🤔

**Comment by markrmiller** [comment]
Yeah, it's my own personal render farm software that runs against a few local pc's and|or spins up RunPod cloud containers, so I have access to everything. Unfortunetly, I don't see any error output for it, nor anything that seems related. The only evidence of it appears to be, in the frames rendered by the master local pc where I work on the blend file, the sky is a light color and on any other local or RunPod machine rendered frames, its black. Frames are rendered the same everywhere, via command line interface against the exact same .blend file.

**Comment by markrmiller** [comment]
Well, after trying everything I can think of, it looks like I have to bite the bullet here and sync the plugin to each node and use the python api to install and enable it.

**Comment by ReinaVV** [comment]
No recent activity, different aproach in newer builds.
Closing to keep the tracker tidy.
